drivers/dma/dma_intel_adsp_gpdma.c

```diff
@@ -234,6 +243,12 @@ int intel_adsp_gpdma_init(const struct device *dev)
 #ifdef CONFIG_SOC_SERIES_INTEL_ACE
 	/* Power up */
 	ret = intel_adsp_gpdma_enable(dev);
+
+	if (ret == 0) {
+		pm_device_init_suspended(dev);
+		ret = pm_device_runtime_enable(dev);
+	}
```

Comment:
Just to be clear that after this, dev will be with the state suspended right ? you have to call `pm_device_runtime_get()` in order to have it set to ACTIVE. After a well succeed `pm_device_runtime_enabled` the device state is suspended.

Reply:
Flow is intended, I want to avoid call of pm_action at this point. Setting device to suspend here is allowing to skip it, when I'm enabling pm runtime.